pghoard/pghoard.py

```diff
@@ -164,7 +193,7 @@ def create_basebackup(self, site, connection_info, basebackup_path, callback_que
         pg_version_server = self.check_pg_server_version(connection_string, site)
         if not self.check_pg_versions_ok(site, pg_version_server, "pg_basebackup"):
             if callback_queue:
-                callback_queue.put({"success": False})
+                callback_queue.put(CallbackEvent(True))
```

Comment:
nit: using keyword argument would be more clear (I'm surprised it works without it though).